app/assets/stylesheets/views/_search.scss

```diff
@@ -135,8 +135,9 @@ main.search {
       }
 
       mark {
-        background-color: $focus-colour;
+        background-color: inherit;
```

Comment:
If the `mark` is inside the `a` the `a`’s background colour will show through the transparent `mark`. Unless I’m really misunderstanding things?
It doesn’t really matter either way, but transparent seems more accurate to what we want to do. E.g. if you had a gradient on the `a` on top of the background colour then the inherited background colour on `mark` would obscure the gradient on focus.